### PR TITLE
Enable scrollable tabs fling scrolling

### PR DESCRIPTION
**Pull request by HansMuller**
Enable fling scrolling in TabBar.

Also changed the TabBar scrollable: parameter to isScrollable: and used Keys to distinguish the different TabNavigators in the tabs.dart example.

### CODE REVIEW

File: sky/sdk/lib/widgets/tabs.dart [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 import 'dart:math' as math;
 
+import 'package:newton/newton.dart';
+//import 'package:newton/friction_simulation.dart';
```
Comment (abarth) [inline sky/sdk/lib/widgets/tabs.dart]: Please don't check in commented out code.  :)

Comment (HansMuller) [inline sky/sdk/lib/widgets/tabs.dart]: // Oops.

File: sky/sdk/lib/widgets/tabs.dart [hunk 2]
```diff
@@ -345,32 +347,73 @@ class Tab extends Component {
...
+  double x(double time) {
+    return super.x(time).clamp(_minScrollOffset, _maxScrollOffset);
+  }
+
+  bool isDone(double time) {
+    return super.isDone(time) ||
+      (x(time) - _minScrollOffset).abs() < tolerance.distance ||
+      (x(time) - _maxScrollOffset).abs() < tolerance.distance;
+  }
+}
```
Comment (abarth) [inline sky/sdk/lib/widgets/tabs.dart]: This doesn't seem like the right place for this code.  Maybe this should be in scroll_behavior or in newton, perhaps named BoundedFrictionSimuation.

Comment (HansMuller) [inline sky/sdk/lib/widgets/tabs.dart]: I've renamed it.

I suppose it belongs in Newton. That will take two steps: get it uulled into Newton and then update DEPS.

Comment (abarth) [inline sky/sdk/lib/widgets/tabs.dart]: Yep.  Both those steps should be easy.

File: DEPS [hunk 3]
```diff
@@ -31,7 +31,7 @@ vars = {
   'cassowary_dart_revision': '7e5afc5b3956a18636d5b37b1dcba1705865564b',
   'collection_dart_revision': '79ebc6fc2dae581cb23ad50a5c600c1b7dd132f8',
   'crypto_dart_revision': 'd4558dea1639e5ad2a41d045265b8ece270c2d90',
-  'newton_dart_revision': '9fbe5fbac809246f7ace4176feca13bdf731e393',
+  'newton_dart_revision': '11cd659e1650591402b69f91b9c4d2a0841fd5bd',
```
Comment (abarth) [inline DEPS]: Can you update //sky/sdk/pubspec.yaml as well so that when we release this code pub will pull in the correct version?

**Comment by abarth** [comment]
LGTM

Please squash your commits before landing.

**Pull request merged**